src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -265,18 +269,6 @@ private IEnumerable<IssueMarker> RemoveSuppressedIssues(IEnumerable<IssueMarker>
             return filteredIssues.OfType<IssueMarker>().ToArray();
         }
 
-        private void RefreshIssues()
-        {
-            UpdateIssues(Factory.CurrentSnapshot.IssueMarkers ?? Enumerable.Empty<IssueMarker>());
```

Comment:
`IssueMarkers` is always set to a value so the new code doesn't have an equivalent of the null-coalesce condition (and in fact it wasn't necessary in the old version of the code either).